src/Components/Form.js

```diff
@@ -71,8 +71,8 @@ class Form extends React.Component {
     let service_id = "default_service";
     let template_id = "tacofeedback";
     let user_id = "user_tDJDGDhY36xUilJYVFzi4";
-    const mailformat = /^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/;
-
+    // const mailformat = /^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/;
```

Comment:
Please ensure that comments are not code. It leaves the code cleaner

Reply:
Thanks @RonKbS 
About the new layout i designed, i was following the new wireframes that @Ugerpaul17 came up with. He shared the images of the wireframes in the taco channel via odoo.
Otherwise, let me look through all the code to remove the comments that are inappropriate.